source/copilot-schema.txt

```diff
@@ -0,0 +1,63 @@
+.. _vsce-copilot-schema:
+
+===============
+/schema Command
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+The ``/schema`` command provides information about your collection
+schema. The |copilot| accesses your connected MongoDB deployment to
+provide schema information in the Copilot generated response. 
+
+Examples
+--------
+
+Provide a Sample Document
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Consider the ``users`` collection in the `Mflix Sample Database
+<https://www.mongodb.com/docs/atlas/sample-data/sample-mflix/#sample_mflix.users>`__.
+If you are connected to a MongoDB deployment that contains this
```

Comment:
Question: What about readers who don't have that sample? I checked that page, and it doesn't have an actual command to create the collection if they don't have it.

Reply:
I added a sample document to illustrate the collection schema. Because this is a UI example as opposed to a code example, I would recommend using the sample database to ensure consistency over having the user be able to replicate the exact example environment